webpack.common.js

```diff
@@ -93,12 +93,17 @@ module.exports = {
         test: /\.scss$/i,
         exclude: /node_modules/,
         use: [
-          MiniCssExtractPlugin.loader,
+          {
+            loader: MiniCssExtractPlugin.loader,
+            options: {
+              publicPath: '../',
+            },
+          },
```

Comment:
@coreylafferty Try adding this to your webpack.common.js in GUS5 and see if it fixes the font path issue

Reply:
@kmonahan that did it! Thank you!